copyright/copyright.js

```diff
@@ -0,0 +1,106 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+
+const copyright = {
+    items: [],
+    missingHeaders: [],
+    copyrightText: '',
```

Comment:
This assumes that the glob(s) you pass will only ever be of the same type. It might be better to keep a dict of extension -> header text so that you can evaluate on a per-file basis which header you need. 

Given that we'll probably only have a handful of file types supported, you could read those in before you start and have them handy throughout the run.